addons/website/models/ir_module_module.py

```diff
@@ -197,6 +198,7 @@ def _post_copy(self, old_rec, new_rec):
                 continue
             old_field = old_rec._fields[src_fname]
             old_translations = old_field._get_stored_translations(old_rec)
+            old_translations = {lang: value for lang, value in old_translations.items() if lang in installed_langs or lang == 'en_US'}
```

Comment:
Just for the sake of readability
```suggestion
            old_translations = {
                lang: value
                for lang, value in old_field._get_stored_translations(old_rec).items()
                if lang in installed_langs or lang == 'en_US'
            }
```